package.json

```diff
@@ -38,14 +38,12 @@
     "@vx/voronoi": "^0.0.196",
     "amplitude-js": "^7.4.3",
     "aws-sdk": "^2.758.0",
-    "bluebird": "^3.7.2",
```

Comment:
While I was here I did some random pruning of modules we don't actually use (bluebird, downshift, tar)